spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/child/WebConfig.java

```diff
@@ -27,23 +26,19 @@
 //@ImportResource({ "classpath:prop.xml" })    
```

Comment:
Same as comment above

Reply:
It must be a consequence of a copy-paste from the ServiceConfig.class. I am removing them as well.